### PR TITLE
chore: add npm metadata

### PR DESCRIPTION
**Pull request by CrabNejonas**
Adds metadata for npm & enables rust crate to be published

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 [package]
 name = "file-icons"
-authors = ["CrabNebula <hello@crabnebula.dev>"]
+authors = ["Jonas Kruckenberg <jonas@crabnebula.dev>", "CrabNebula <hello@crabnebula.dev>"]
 edition = "2021"
 version = "0.1.0"
```
Comment (lucasfernog-crabnebula) [inline Cargo.toml:5]:
```suggestion
version = "0.0.0"
```

File: .changes/metadata.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"file-icons": patch
+---
+
+Update manifest metadata
```
Comment (lucasfernog-crabnebula) [inline .changes/metadata.md:5]:
```suggestion
---
"file-icons": minor
---

Initial release.
```

**Review by lucasfernog-crabnebula — CHANGES_REQUESTED** [review]

**Comment by lucasfernog-crabnebula** [comment]
Btw when we merge this one, we need to have the crate ready to be published otherwise we'll be stuck for NPM releases.

**Comment by CrabNejonas** [comment]
> Btw when we merge this one, we need to have the crate ready to be published otherwise we'll be stuck for NPM releases.

yeah let's close this and focus on the npm changes

**Comment by CrabNejonas** [comment]
superseded by #15

**Pull request closed without merging**